frontend/packages/rhoas-plugin/src/catalog/providers/useRhoasCatalog.tsx

```diff
@@ -81,39 +81,42 @@ const useRhoasCatalog: CatalogExtensionHook<CatalogItem[]> = ({
     ];
 
     if (isServiceAccountValid) {
-      const {
-        serviceName,
-        name,
-        type,
-        uid,
-        description,
-        provider,
-        tags,
-        icon,
-        ctaLabel,
-        details,
-      } = RHOASService;
-
-      const rhoasCard: CatalogItem[] = [
-        {
+      const rhoasCard: CatalogItem[] = [];
+      RHOASServices.forEach(rhoasService=>{
```

Comment:
intentionally made some ineffective implenentation so we can improve it by using map